tests/chart_tests/test_houston_api_deployment.py

```diff
@@ -21,15 +22,39 @@ def test_houston_api_deployment(self, kube_version):
         doc = docs[0]
         assert doc["kind"] == "Deployment"
         assert "annotations" not in doc["metadata"]
+        assert {
+            "tier": "astronomer",
+            "component": "houston",
+            "release": "release-name",
+        } == doc["spec"]["selector"]["matchLabels"]
+
         assert (
-            {
-                "tier": "astronomer",
-                "component": "houston",
-                "release": "release-name",
-            }
-            == doc["spec"]["selector"]["matchLabels"]
-            == doc["spec"]["template"]["metadata"]["labels"]
+            "app" in doc["spec"]["template"]["metadata"]["labels"]
+            and "houston" in doc["spec"]["template"]["metadata"]["labels"]["app"]
         )
+        assert (
+            "tier" in doc["spec"]["template"]["metadata"]["labels"]
+            and "astronomer" in doc["spec"]["template"]["metadata"]["labels"]["tier"]
+        )
```

Comment:
Probably easier to grok, and more concise.

```suggestion
        assert doc["spec"]["template"]["metadata"]["labels"].get("app") == "houston"
```